bench/FileConnectionBench.java

```diff
@@ -0,0 +1,48 @@
+package benchmark;
+
+import com.sun.cldchi.jvm.JVM;
+import com.sun.midp.crypto.SecureRandom;
+import java.io.OutputStream;
+import java.util.Random;
+import javax.microedition.io.Connector;
+import javax.microedition.io.file.FileConnection;
+import org.mozilla.MemorySampler;
+
+public class FileConnectionBench {
+  void runBenchmark() {
+    try {
+      long start, time;
+
+      start = JVM.monotonicTimeMillis();
+
+      String privateDir = System.getProperty("fileconn.dir.private");
+      Random random = new Random();
+      String filename = new javax.bluetooth.UUID(java.lang.Math.abs(random.nextInt())).toString();
```

Comment:
javax.bluetooth.UUID only exists if you build with JSR_082=1, could you use another name?
Even a static name would work, with a:
```Java
        if (file.exists()) {
            file.delete();
        }
```